src/FondOfSpryker/Yves/CheckoutPage/Process/Steps/SuccessStep.php

```diff
@@ -80,12 +80,10 @@ public function execute(Request $request, AbstractTransfer $quoteTransfer)
             $this->cartClient->clearQuote();
         }
 
-        if (method_exists($quoteTransfer->getPayment(), 'getPayone')) {
+        if ($quoteTransfer instanceof QuoteTransfer) {
             $this->quoteTransfer = $quoteTransfer;
         }
```

Comment:
Is this still necessary? Doesn't seem to do anything at all.

Reply:
i think it is ok. the method expects as parameter a AbstractTransfer, so i have to make sure that it is really a QuoteTransfer. 

But i am a little bit confused why we do not return the given QuoteTransfer instead creating an new on?!

Reply:
That's why I'm asking.. it seems that this doesn't make any sense.